fang/src/asynk/async_runnable.rs

```diff
@@ -40,7 +32,7 @@ impl From<SerdeError> for FangError {
 /// Implement this trait to run your custom tasks.
 #[typetag::serde(tag = "type")]
 #[async_trait]
-pub trait AsyncRunnable: Send + Sync {
+pub trait AsyncRunnable: Send + Sync + Send {
```

Comment:
send is duplicated twice

Reply:
ups

Reply:
fixed here [store uuid as text , so this way PostgreSQL schema will not change](https://github.com/ayrat555/fang/pull/141/commits/5119dcf870a991ef2fcbbdcb0b471dec92739f89)